utils/build.ps1

```diff
@@ -3314,15 +3338,18 @@ if (-not $SkipBuild) {
   Invoke-BuildStep Build-BuildTools $BuildPlatform
   if ($IsCrossCompiling) {
     Invoke-BuildStep Build-XML2 $BuildPlatform
-    Invoke-BuildStep Build-Compilers $BuildPlatform
+    Invoke-BuildStep Build-Compilers $BuildPlatform -Variant $PinnedToolchainVariant
   }
   if ($IncludeDS2) {
     Invoke-BuildStep Build-RegsGen2 $BuildPlatform
   }
 
   Invoke-BuildStep Build-CMark $HostPlatform
   Invoke-BuildStep Build-XML2 $HostPlatform
-  Invoke-BuildStep Build-Compilers $HostPlatform
+  Invoke-BuildStep Build-Compilers $HostPlatform -Variant "Asserts"
+  if ($IncludeNoAsserts) {
+    Invoke-BuildStep Build-Compilers $HostPlatform -Variant "NoAsserts"
+  }
```

Comment:
I wonder if we should build both toolchains first or just the asserts toolchain first and then build the additional variants at the end.

Reply:
Right this question originates from the idea of development. Is it better to get an idea that the toolchain and libraries build before we build the toolchain or is it better to build the toolchain both ways before building the libraries and tools?

Reply:
Moved it to the bottom